src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpSimplifyTypeNamesDiagnosticAnalyzer.cs

```diff
@@ -149,18 +138,15 @@ internal static bool CanSimplifyTypeNameExpression(SemanticModel model, SyntaxNo
                 else if (expression.Kind() == SyntaxKind.SimpleMemberAccessExpression)
                 {
                     var memberAccess = (MemberAccessExpressionSyntax)expression;
-                    diagnosticId = memberAccess.Expression.Kind() == SyntaxKind.ThisExpression ?
-                        IDEDiagnosticIds.RemoveQualificationDiagnosticId :
-                        IDEDiagnosticIds.SimplifyMemberAccessDiagnosticId;
+                    diagnosticId = memberAccess.Expression.Kind() == SyntaxKind.ThisExpression
+                        ? IDEDiagnosticIds.RemoveQualificationDiagnosticId 
```

Comment:
> ?  [](start = 23, length = 3)

Can we have some kind of code style enforcement for this so Cyrus doesn't have to track all these down himself?